doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -663,6 +633,129 @@ Reserved keywords
    section are examples of user-defined variables (e.g. ``inCollection``) and
    are not required by BPS.
 
+.. _managing-job-memory:
+
+Managing job memory requirements
+--------------------------------
+
+The primary parameters controlling job memory requirements is called
+``requestMemory``.  It specifies the amount of memory (in MiB) each job will
+have at its disposal during the execution of the workflow. Its default value
+is 2048 (2 GiB).
+
+The default value can be overridden globally for all the jobs as well as for
+jobs running specific tasks.  For example, including the lines below in your
+BPS config will result in all jobs having 4 GiB of memory during their
+execution *except* jobs running ``skyCorr`` task that will be able to use up to
+8 GiB of RAM as task specific setting takes precedence over the global
+one if both are present in your BPS config.
+
+.. code-block::
+
+   requestMemory: 4096
+
+   pipetasks:
+     skyCorr:
+       requestMemory: 8192
+
+.. _automatic-memory-scaling:
+
+Automatic memory scaling
```

Comment:
I don't know what folks will search for.  For example is "Automatic retries with increased memory" better?

Reply:
I'll include this phrase in the paragraph opening the section.